### PR TITLE
Create social media login

### PR DESCRIPTION
**Pull request by irnafitriani**

### CODE REVIEW

**Comment by WendySanarwanto** [comment]
## Feedbacks for @irnafitriani:
- Comment ilne 21 to 24 in gradle.properties file. This will prevent build error on machines that are not connected to bandungisa3
- iOS: Error occurs when touching "sign in with facebook" button on iPhone. ios version: 9.3.2. See this following screenshot:
  image

**Comment by raisarijasa** [comment]
Comment line 21 to 24 in android/gradle.properties file. This will prevent build error on machines that are not connected to bandungisa3

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -15,6 +15,20 @@
...
+        android:name="com.facebook.FacebookActivity"
+        android:configChanges="keyboard|keyboardHidden|screenLayout|screenSize|orientation"
+        android:label="@string/app_name"
+        android:theme="@android:style/Theme.Translucent.NoTitleBar" />
+        
+      <meta-data 
+      android:name="com.facebook.sdk.ApplicationId" 
+      android:value="@string/facebook_app_id"/>
+      
+      <activity
```
Comment (raisarijasa) [inline android/app/src/main/AndroidManifest.xml:29]: Duplicate code. Line 29 and 42. Please remove one of them

File: apps/modules/login/screen_login.js [hunk 2]
```diff
@@ -1,91 +1,189 @@
...
-        Alert.alert('Login Failed', JSON.stringify(error));
-        console.log(`[Error] - Sign in attempt is failing. Error: ${JSON.stringify(error)}`);
-      })
-      .done();
+          this.setState({
+            loading: false
+          });
+        })
+        .catch(error => {
+          React.AlertIOS.alert(
```
Comment (raisarijasa) [inline apps/modules/login/screen_login.js]: Please use Alert instead of AlertIOS

**Comment by raisarijasa** [comment]
Please display ID, Access Token, and Public profile in Alert when login using Social Media

**Pull request merged**